app/assets/stylesheets/welcome.scss

```diff
@@ -0,0 +1,63 @@
+
+
+video {
+  position: fixed;
+  top: 50%;
+  left: 50%;
+  min-width: 102%;
+  min-height: 100%inherit;
+  margin: 50px 0 0 0;
```

Comment:
Shorthand form for property `margin` should be written more concisely as `50px 0 0` instead of `50px 0 0 0`